src/geojson/geojson.ml

```diff
@@ -310,12 +322,13 @@ module Make (J : Intf.Json) = struct
               "A Geojson feature requires the type `Feature`. No type was \
                found.")
 
-    let to_json ?bbox (t, props) =
+    let to_json { id; geometry; properties } ?bbox t =
```

Comment:
```suggestion
    let to_json ?bbox { id; geometry; properties } =
```

Reply:
This leaves t inbounded sir and when t is replace from Geometry.to_json t to Geometry.to_json geometry opens an error on line 450